src/jaegertracing/Constants.h.in

```diff
@@ -31,6 +31,7 @@ static constexpr auto kSamplerTypeTagKey = "sampler.type";
 static constexpr auto kSamplerParamTagKey = "sampler.param";
 static constexpr auto kTraceContextHeaderName = "uber-trace-id";
 static constexpr auto kTracerStateHeaderName = kTraceContextHeaderName;
+static constexpr auto kW3CTraceContextHeaderName = "traceparent";
```

Comment:
I take it you're only supporting `traceparent`, but not `tracestate`. This makes it non-compliant propagator. At minimum it should pass on the `tracestate` unchanged, as was done in the Java client https://github.com/jaegertracing/jaeger-client-java/blob/5831d9ed7b465971d6c5211e3f56371cc083541a/jaeger-core/src/main/java/io/jaegertracing/internal/propagation/TraceContextCodec.java

Reply:
I will look into it

Reply:
done